evergreen.yml

```diff
@@ -0,0 +1,123 @@
+#######################################
+#              Parameters             #
+#######################################
+parameters:
+  - key: hash
+    value: a91ffb61aeae8e934d50f351155c88a75e1cb614
+    description: default value until I can figure out Inter-project Dependencies
+  - key: version
+    value: 0.13
+    description: default value, should be passed in via command line or Inter-project
+
+#######################################
+#              Functions              #
+#######################################
+functions:
+  get-bins:
+    command: subprocess.exec
+    params:
+      command: ./unpackBins.sh ${hash}
+      working_dir: curatorbin/curatorbin
+
+  upload-to-testpypi:
+    - command: subprocess.exec
+      params:
+        command: pip3 install -q build twine
+        working_dir: curatorbin
+    - command: subprocess.exec
+      params:
+        command: python3 -m build
+        working_dir: curatorbin
+    - command: subprocess.exec
+      params:
+        command: python3 -m twine upload --repository testpypi -u __token__ -p ${testpypi_pwd} dist/*
+        working_dir: curatorbin
+
+#######################################
+#                Tasks                #
+#######################################
+tasks:
+
+  - name: publish-to-testpypi
+    commands:
+      - command: git.get_project
+        params:
+          directory: curatorbin
+      - func: get-bins
+      - func: upload-to-testpypi
+
+  - name: test-from-pypi
+    commands:
+      - command: git.get_project
+        type: setup
+        params:
+          directory: curatorbin
+      - command: subprocess.exec
+        params:
+          command: pip3 install pytest
+      - command: subprocess.exec
+        params:
+          command: pip3 install -vvv -i https://test.pypi.org/simple/ curatorbin==${version}
+      - command: subprocess.exec
+        params:
+          command: python3 -m pytest
+          working_dir: curatorbin/testdir
+    depends_on:
+      - name: publish-to-testpypi
+        variant: ubuntu2004
+        status: "*"
+
+  - name: test-from-pypi-windows
+    commands:
+      - command: git.get_project
+        type: setup
+        params:
+          directory: curatorbin
+      - command: subprocess.exec
+        params:
+          command: pip3 install pytest
+      - command: subprocess.exec
+        params:
+          command: pip3 install -vvv -i https://test.pypi.org/simple/ curatorbin==${version}
+      - command: subprocess.exec
+        params:
+          command: 'c:\\python39\\python.exe -m pytest'
+          working_dir: curatorbin/testdir
+    depends_on:
+      - name: publish-to-testpypi
+        variant: ubuntu2004
+        status: "*"
+
+#######################################
+#           Buildvariants             #
+#######################################
+buildvariants:
+
+  - name: ubuntu2004
+    display_name: Ubuntu 20.04
+    run_on:
+      - ubuntu2004-small
```

Comment:
Should we add more variants here and for the macos?

Reply:
1604 doesn't work and I don't want to tempt fate. I can add some more for the macos